heroku-24-build/setup.sh

```diff
@@ -53,6 +53,7 @@ packages=(
   patchelf
   python3 # Often needed during the building of non-Python apps. e.g. For Node.js packages that use node-gyp.
   zlib1g-dev
+  libtirpc3t64 # used to be a dependency of iproute2 (e.g. /bin/ip linked against libtirpc.so.3), now requiring explicitly to prevent package removal - can be removed for heroku-26
```

Comment:
```suggestion
  libtirpc3t64 # used to be a dependency of iproute2 (e.g. /bin/ss linked against libtirpc.so.3), now requiring explicitly to prevent package removal - can be removed for heroku-26
```

Had the wrong binary there, sorry ;)

Reply:
Fixed in https://github.com/heroku/base-images/pull/357/commits/24806357ef78332b24fa731de07876b51538f964